buildtools/fslex/fslex.fsproj

```diff
@@ -5,7 +5,8 @@
     <TargetFramework>$(FSharpNetCoreProductTargetFramework)</TargetFramework>
     <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
     <RollForward Condition="'$(BUILDING_USING_DOTNET)' == 'true'">LatestMajor</RollForward>
-    <NoWarn>$(NoWarn);64;1182;1204</NoWarn>         <!--Temporary fix for sourcebuild -->
+    <NoWarn>$(NoWarn);64;1204;1182</NoWarn>         <!--Temporary fix for sourcebuild -->
+    <OtherFlags>$(OtherFlags) --warnaserror-:1182</OtherFlags>
```

Comment:
I'd suggest using `WarningsAsErrors` for uniformity with other places.

Reply:
WarningsAsErrors  is equal to `--warnaserror`, `--warnaserror-` is inverse (note the minus at the end) and is needed to override properties that can cascade from parent folders etc.


I will test if we can start using a new property (making sure it can also be used within Proto) for this inverse. If possible, this will go in a separate PR.

Reply:
Ah right. Sure, that's okay then.